src/plugin.py

```diff
@@ -115,11 +116,16 @@ async def get_game_communication_ids(self, title_ids: List[TitleId]) -> Dict[Tit
 
     async def get_subscriptions(self) -> List[Subscription]:
         is_plus_active = await self._psn_client.get_psplus_status()
-        return [Subscription(subscription_name="PlayStation PLUS", end_time=None, owned=is_plus_active)]
+        return [Subscription(PLAYSTATION_PLUS, is_plus_active, None),
+            Subscription(PLAYSTATION_NOW, None, None, SubscriptionDiscovery.USER_ENABLED)]
```

Comment:
good enough to have it as user-enable experimental feature.

For future I see that PSN web store client uses graphQL API eg. https://web.np.playstation.com/api/graphql/v1/op?operationName=oracleUserProfile
and there is something like
```json
"data": {
"subscriptionState": [
    {"__typename": "Subscription", "subscriptionStatus": "NEVER", "subscriptionType": "PSPLUS"}
    {"__typename": "Subscription", "subscriptionStatus": "NEVER", "subscriptionType": "PSNOW"}
    {"__typename": "Subscription", "subscriptionStatus": "NEVER", "subscriptionType": "EAACCESS"}
]}
```
but I'm not sure if our token has access there